docs/quickstart.rst

```diff
@@ -1,14 +1,48 @@
 Quick start
 ===========
 
-Install using `nodejs <http://nodejs.org>`_::
+Installation 
+------------
+
+First, install `git <https://git-scm.com/downloads>`_. 
+
+Second, install `Node.js <http://nodejs.org>`_. Webppl is written in JavaScript, and requires Node to run. After it is installed, you can use npm (**n**\ ode **p**\ ackage **m**\ anager) to install webppl::
```

Comment:
I *think* "WebPPL" is used more often in the docs at the moment, so I'd use that for now. It also means we can punt on whether or not "webppl" should be "Webppl" at the beginning of a sentence...